lib/geminabox/server.rb

```diff
@@ -107,7 +108,9 @@ def dependency_cache
 
     get '/reindex' do
       serialize_update do
-        self.class.reindex(:force_rebuild)
+        params[:force_rebuild] ||= 'true'
+        force_rebuild = params[:force_rebuild] == 'true'
```

Comment:
It should still default to true correct? That will maintain default behavior.

As for validation, I'll add it but I'd prefer to use 400 as the status code as we're making this a bad request not a missing resource.